### ISSUE TITLE
Warning messages using \copyright

### ISSUE THREAD

**Issue by stdedos** [latex3/hyperref#178]
With text:
```latex
% View: Two Page, Cover Solo / With Bookmarks - Also set PDF Metadata
\hypersetup{unicode=true, pdfpagelayout=TwoPageRight, pdfpagemode=UseOutlines,
	pdfdisplaydoctitle=true, colorlinks, linkcolor={red!50!black},
	citecolor={blue!50!black}, urlcolor={blue!80!black},% hidelinks
	pdfinfo={
		Title={Thesis of \AuthorName{}},
		Creator={\AuthorName{} using *TeX},
		Producer={pdfTeX 3.14159265-2.6-1.40.17},
		Author={\AuthorName{}},
		Subject={\ThesisTitle{}},
		Keywords={TODO, \AlgoName{}, ...}
	},
	pdfcopyright={\CopyrightText{} All rights reserved.}
}
```

with `\CopyrightText` being:
```latex
\newcommand{\CopyrightText}{Copyright \copyright\; 2016-2017 by \AuthorName{}.}
```

gives

```latex
Package hyperref Warning: Token not allowed in a PDF string (Unicode):
(hyperref)                removing `\leavevmode@ifvmode' on input line 166.


Package hyperref Warning: Token not allowed in a PDF string (Unicode):
(hyperref)                \kern 3.26286pt
(hyperref)                removed on input line 166.
```

Not even

```latex
\newcommand{\CopyrightText}{Copyright \texorpdfstring{\copyright}{(c)}\; 2016-2017 by \AuthorName{}.}
```

helps!

Only setting
```latex
%	pdfcopyright={\CopyrightText{} All rights reserved.}
```
fixes the error

**Comment by stdedos** [comment]
Oops! `pdfcopyright` is provided by https://www.ctan.org/pkg/hyperxmp and not `hyperref`!

**Comment by u-fischer** [comment]
The problem is the `\;` Use 

     \newcommand{\CopyrightText}{Copyright \copyright\texorpdfstring{\;}{ } 2016-2017 by \AuthorName{}.}

**Comment by muzimuzhi** [comment]
more or less a duplicate of #176

Another workaround by using `\pdfstringdefDisableCommands`, as documented in `texdoc hyperref`, sec. 4.1:

```tex
\documentclass{article}
\usepackage{hyperref}

\pdfstringdefDisableCommands{%
  \let\leavevmode@ifvmode\@empty
}

\hypersetup{
  pdftitle={\;}
}

\begin{document}
content
\end{document}
```

**Comment by davidcarlisle** [comment]
or (more normally) use `\ ` rather than `\;` for the space.

**Comment by stdedos** [comment]
> or (more normally) use `\ ` rather than `\;` for the space.

I think I wanted a variation of forced space (longer I presume, I don't remember anymore)

> more or less a duplicate of #176

I saw it actually! I didn't guess `\;` was producing `\leavevmode@ifvmode` (and then there was `\kern 3.26286pt` too). With "Unicode", only `\copyright` made sense to me; but some PDF viewers actually rendered that correctly.

I went with the first suggestion; thank you everyone anyway for your inputs 😀
